dependencies.gradle

```diff
@@ -77,6 +80,12 @@ ext.libraries = [
         retrofitRxJavaAdapter   : "com.squareup.retrofit2:adapter-rxjava:$versions.retrofit",
         gson                    : "com.google.code.gson:gson:$versions.gson",
 
+        rxJavaProguardRules     : "com.artemzin.rxjava:proguard-rules:$versions.rxJavaProguardRules",
+        retrolambdaProguardRules: "com.github.yongjhih.android-proguards:retrolambda-pro:$versions.proguardRules",
+        retrofitProguardRules   : "com.github.yongjhih.android-proguards:retrofit2-pro:$versions.proguardRules",
```

Comment:
Mmmmmmm, wait, they all have one version? This is super weird and if so — I'm voting against these libs